src/main/java/org/candlepin/model/DeletedConsumerCurator.java

```diff
@@ -117,6 +121,98 @@ public int countByConsumerUuid(String uuid) {
             .intValue();
     }
 
+    @Transactional
```

Comment:
nack: individual methods should not be marked as transactional. Transaction boundaries should be defined by the scope of an externally facing operation, such as a job or API request. 

I know we still have a number of these throughout the code base, but the goal is to eventually eliminate them all in favor of transaction boundaries as close to user operations as possible.